core/include/io/Filter_File_Handle.hpp

```diff
@@ -19,16 +20,19 @@ namespace IO
 	class Filter_File_Handle
 	{
 	private:
-		std::unique_ptr<std::ifstream> myfile;
-		std::string filename;
 		std::size_t found;
 		std::string line;
+        std::string comment_tag;
 		std::string dump;
-
 	public:
+        IO::VF_FileFormat ff;
+        std::string filename;
+        std::unique_ptr<std::ifstream> myfile;
 		std::istringstream iss;
+        
 		// Constructs a Filter_File_Handle with string filename
-		Filter_File_Handle(const std::string& s);
+		Filter_File_Handle( const std::string& s, 
+                            IO::VF_FileFormat format = VF_FileFormat::SPIRIT_GENERAL );
```

Comment:
Should the `Filter_File_Handle` really care about the format of the file? I suppose it should be more abstract and simply be created with appropriate comment strings etc.